modules/videoio/src/cap_ffmpeg_impl.hpp

```diff
@@ -1428,8 +1429,10 @@ bool CvCapture_FFMPEG::grabFrame()
 {
     bool valid = false;
 
-    int count_errs = 0;
-    const int max_number_of_attempts = 1 << 9;
+    static const size_t max_stream_attempts = cv::utils::getConfigurationParameterSizeT("OPENCV_FFMPEG_STREAM_ATTEMPTS", 4096);
```

Comment:
`OPENCV_FFMPEG_READ_STREAM_ATTEMPTS`

Reply:
Since it is `av_read_frame` maybe `READ_FRAME_ATTEMPTS` would fit better? Or just `READ_ATTEMPTS` for short?